stubs/tensorflow/@tests/stubtest_allowlist.txt

```diff
@@ -15,8 +15,49 @@ tensorflow.DType.__getattr__
 tensorflow.Graph.__getattr__
 tensorflow.Operation.__getattr__
 tensorflow.Variable.__getattr__
+tensorflow.keras.layers.Layer.__getattr__
 # Internal undocumented API
 tensorflow.RaggedTensor.__init__
 # Has an undocumented extra argument that tf.Variable which acts like subclass
 # (by dynamically patching tf.Tensor methods) does not preserve.
 tensorflow.Tensor.__getitem__
+# stub internal utility
+tensorflow._aliases
+
+# Tensorflow imports are cursed.
+# import tensorflow.initializers
+# import tensorflow as tf
+# tf.initializers
+# Usually these two ways are same module, but for tensorflow the first way
+# often does not work and the second way does. The documentation describes
+# tf.initializers as module and has that type if accessed the second way,
+# but the real module file is completely different name (even package) and dynamically handled.
+# tf.initializers at runtime is <module 'keras.api._v2.keras.initializers' from '...'>
+tensorflow.initializers
+
+# Layer constructor's always have **kwargs, but only allow a few specific values. PEP 692
```

Comment:
I don't think we have PEP 692 tracker ticket yet. Will make one later if no one covers it first.

Reply:
I think PEP 692 might already have sufficient support to use it; at least mypy and pyright support it. Haven't checked pytype though.

Reply:
Testing basic 692 support:

```python
class Foo(TypedDict, total=False):
    a: int
    b: str


def f(**kwargs: Unpack[Foo]) -> None:
    ...


f(a=1, b="2")  # OK
f(a=1, b=2)  # Error: b has type str
f(a=1, b="2", c=3)  # Error: unexpected keyword argument "c"
```

- pyright works fine
- mypy works if we add --enable-incomplete-feature=Unpack (otherwise it gives `error: "Unpack" support is experimental, use --enable-incomplete-feature=Unpack to enable`)
- pytype produces `typing_extensions.Unpack not supported yet [not-supported-yet]`. No errors otherwise.
- pyre is same as pytype and just produces `pep_692.py:11:16 Undefined or invalid type [11]: Annotation `Unpack` is not defined as a type.`

I'll open ticket to track.